libs/platform/src/lib/components/button/button.module.ts

```diff
@@ -8,4 +8,4 @@ import { ButtonComponent } from './button.component';
     exports: [ButtonComponent],
     declarations: [ButtonComponent]
 })
-export class PlatformButtonModule {}
+export class PlatformButtonModule { }
```

Comment:
no need to include this file as it does not change anything

Reply:
Prettifier suggested having a space inside {}. By which it shows in diffs.